### ISSUE TITLE
Generation is always the same

### ISSUE THREAD

**Issue by davidmrdavid** [deepsound-project/samplernn-pytorch#14]
Hello,

I'm running train.py with different datasets and, although the loss decreases differently for each dataset, the generated samples are always exactly the same across different datasets. Is this a known issue?

**Comment by davidmrdavid** [comment]
It turns out that the issue had to do with caching on my end. Closing the issue
